config-service-impl/src/main/java/org/hypertrace/config/service/store/DocumentConfigStore.java

```diff
@@ -233,7 +245,20 @@ public List<ContextSpecificConfig> getAllConfigs(
 
     configList.sort(
```

Comment:
this is a bug, right? It'll discard the sort applied in mongo. I thought we discussed removing the in memory sort in the impl PR but I don't think I followed up on it